### PR TITLE
Feature/a11y clean html from cms

### PR DESCRIPTION
**Pull request by camilokawerin**
Content coming from CMS includes some malformed HTML, i. e. empty links wich causes duplicated tab stop on links.

imagen

**What kind of change does this PR introduce?**

Added a function from a module for cleaning the HTML in the Content component.

**Does this PR introduce a breaking change?**

Yes, `yarn install` should be run to install the package 'clean-html'.

**What needs to be documented once your changes are merged?**

No documentation is needed.

### CODE REVIEW

File: src/components/Content.js [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
-  <div className={className} dangerouslySetInnerHTML={{ __html: content }} />
-)
+export const HTMLContent = ({ content, className }) => {
+  const [cleanHTML, setCleanHTML] = useState(content);
+  
+  useEffect(() => {
+    const options = {
+      'remove-empty-tags': ['a', 'p'],
+      'indent': '',
+      'break-around-tags': []
```
Comment (mrazzari) [inline src/components/Content.js:12]: I assume the last 2 were added to keep any existing behavior?

File: src/components/Content.js [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+    }
+    cleaner.clean(content, options, cleaned => {
+      console.log(cleaned)
```
Comment (mrazzari) [inline src/components/Content.js:15]: Remove.

File: src/components/Content.js (hunk 1 above)
Comment (camilokawerin) [reply src/components/Content.js:12]: Yes, the final HTML is minimized, so we don't need indentation and lines breaks.

File: src/components/Content.js (hunk 2 above)
Comment (camilokawerin) [reply src/components/Content.js:15]: Damn, sorry.

**Review by mrazzari — APPROVED** [review]
LGTM.

**Review by smarcet — APPROVED** [review]
looks good to me

**Pull request merged**